File: Webtechnologien/Eigenstudium_D/schleifen.py
#!C:\ProgramData\Anaconda3\python.exe
print()
print('<!DOCTYPE html>')
print('<html lang="en">')
print(''' <head> <meta charset="UTF-8">
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <link href="style.css" rel="stylesheet" type="text/css">
  <title>I'M a PRICK</title></head>''')

#body tag
print('<body>')

#For loop
x = 1
print('<table>')
for i in range(3):
  print('<tr>')
  for j in range(5):
    if x%2:
      print('<td class=uneven>{}</td>'.format(x))
    else:
      print('<td class=even>{}</td>'.format(x))
    x+=1
  print('</tr>')
print('</table><br>')

#while loop
x,i = 1,0
print('<table>')
while i<3:
  print('<tr>')
  j=0
  while j<5:
    if x%2:
      print('<td class=uneven>{}</td>'.format(x))
    else:
      print('<td class=even>{}</td>'.format(x))
    x+=1
    j+=1
  print('</tr>')
  i+=1
print('</table><br>')


#there is no do-while loop in python. the easy way would be to execute that thing at least once manually and then initiallize a loop.
#but thats kinda tedious. I'll just leave it here. 
#I am so blown away by this. this is amazing. I didn't know that Apache and XAMPP was capable of doing this. 

print('</body>')
print('</html>')

 

File: SEM1_WS2020/Webtechnologien/Eigenstudium_D/schleifen.php
<!DOCTYPE html>
<html lang="en">

<head>
  <meta charset="UTF-8">
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <link href="https://fonts.googleapis.com/css2?family=Open+Sans&family=PT+Serif&display=swap" rel="stylesheet"> 
  <title>Schleifen</title>
  <style>
    *{
      font-family:"Open Sans";
    }
    table{
      border-style: solid;
      border-color:grey;
      text-align: center;
      font-size: larger;
    }
    td{
      padding:5px;
    }
    .even{
      background-color: lightskyblue;
    }
  </style>

</head>

<body>
  <?php
    echo "<table>";
    $x = 1;
    for ($i = 0; $i < 3; $i++) {
      print("<tr>");
      for ($j = 0; $j < 5; $j++) {

        if ($x % 2) {
          print("<td class ='uneven'>$x</td>");
        } else {
          print("<td class ='even'>$x</td>");
        }
        $x++;
      }
      print("</tr>");
    }
    echo "</table> <br>";
    
    echo "<table>";
    $x = 1;
    $i=0;
    
    while ($i < 3)
    {
      $j=0;
      print("<tr>");
      while($j < 5)
      {
        if ($x % 2) {
          print("<td class ='uneven'>$x</td>");
        } else {
          print("<td class ='even'>$x</td>");
        }
        $x++;
        $j++;
      }
      print("</tr>");
      $i++;
    }
    echo "</table> <br>";

    echo "<table>";
    $x = 1;
    $i=0;
    
    do
    {
      $j=0;
      print("<tr>");
      do
      {
        if ($x % 2) {
          print("<td class ='uneven'>$x</td>");
        } else {
          print("<td class ='even'>$x</td>");
        }
        $x++;
        $j++;
      } while($j < 5);
      print("</tr>");
      $i++;
    } while ($i < 3);
    echo "</table> <br>";
  
    
  ?>
</body>

</html>